UN-OCHA/hpc-cdm
# [PROPOSAL] - HPC-9658 Add cypress tests

## Nature of PR
- Bug-fix
- Hot-fix
- Feature
- 🟢  Testing
- Rewrite
- CI


## Description

Initial implementation of tests for FTS Admin. These test consist of mocked data and mocked endpoints, so we are focusing and isolating frontend behaviors. This is a decision in order to have deterministic tests. Therefore these tests would not really be E2E tests.


## How to test changes

For testing, you can run the test in headed mode by running :
```
npm run e2e hpc-ftsadmin-e2e -- --watch
```
All the test that are written should not fail.

## TODO:

These test can be really improved and implement way more use cases, but I think as a first approach is best to review these small tests and decide if we want to invest more time into building them
